source/isaaclab_rl/isaaclab_rl/rsl_rl/exporter.py

```diff
@@ -47,12 +47,23 @@ def export_policy_as_onnx(
 class _TorchPolicyExporter(torch.nn.Module):
     """Exporter of actor-critic into JIT file."""
 
-    def __init__(self, actor_critic, normalizer=None):
+    def __init__(self, policy, normalizer=None):
         super().__init__()
-        self.actor = copy.deepcopy(actor_critic.actor)
-        self.is_recurrent = actor_critic.is_recurrent
+        # infer policy type
+        if hasattr(policy, "actor"):
+            self.actor = copy.deepcopy(policy.actor)
+        elif hasattr(policy, "student"):
+            self.actor = copy.deepcopy(policy.student)
+        else:
+            raise ValueError("Policy does not have an actor/student module.")
+        self.is_recurrent = policy.is_recurrent
         if self.is_recurrent:
-            self.rnn = copy.deepcopy(actor_critic.memory_a.rnn)
+            if hasattr(policy, "memory_a"):
+                self.rnn = copy.deepcopy(policy.memory_a.rnn)
+            elif hasattr(policy, "memory_s"):
+                self.rnn = copy.deepcopy(policy.memory_s.rnn)
+            else:
+                raise ValueError("Policy does not have a memory module.")
```

Comment:
NIT: Can you first check if it is recurrent and then simplify it to "if actor" and inside "if recurrent". Right now it gets a bit cryptic what is _a and _s